test/execute.js

```diff
@@ -441,7 +441,7 @@ describe('execute', () => {
       })
     })
 
-    it('should handle requestContentType', function () {
+    it('should not add content-type with no form-data or body param', function () {
```

Comment:
I see that. But we shouldn't :stuck_out_tongue: if there is a `requestContentType` we must use it. Its an override provided by the user ( or swagger-ui in our case ).